publication-commons/src/main/java/no/unit/nva/publication/model/FilesApprovalEntry.java

```diff
@@ -57,10 +58,24 @@ public FilesApprovalEntry complete(Publication publication, UserInstance userIns
 
     public FilesApprovalEntry applyPublicationChannelClaim(URI organizationId,
                                                            SortableIdentifier channelClaimIdentifier) {
-        this.setReceivingOrganizationDetails(new ReceivingOrganizationDetails(organizationId, organizationId, channelClaimIdentifier));
+        this.setReceivingOrganizationDetails(
+            new ReceivingOrganizationDetails(organizationId, organizationId, channelClaimIdentifier));
         return this;
     }
 
+    public FilesApprovalEntry clearPublicationChannelClaim(SortableIdentifier channelClaimIdentifier) {
+        if (isUnderClaimedChannelInfluence(channelClaimIdentifier)) {
```

Comment:
Weird wording here, but I know not what any of this means <- also weird wording :D

Reply:
Did some adjustments to the wording.